build/int.cloudbuild.yaml

```
@@ -89,6 +89,21 @@ steps:
     - verify simple-regional-local
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'cft test run TestSimpleRegional --stage teardown --verbose']
+- id: apply simple-regional-additional-local
+  waitFor:
+    - init-all
```

reached limit of simultaneous

```suggestion
    - init-all
    - teardown simple-regional-local
```